src/js/helpers/input-trailing-digits-count.js

```diff
@@ -0,0 +1,17 @@
+import config from '@/config'
+
+export function inputStepByDigitsCount (count) {
```

Comment:
I believe we can use custom vuelidate’s validator for this. Or even better maybe we can modify the existing one?

Reply:
Also there’s no CHANGELOG notes about this change

Reply:
I think we can modify default `minValue` via passing not a value but object with two supported keys: `minValue` or `minValueByDigitsCount`. In the validator, we can check for these keys and return validation rule depending on the passed key. What do you think? We need to fix `minValue` usage throughout our app in the result

Or we can just add a new validation rule – `minValueByDigitsCount`.

Reply:
Ok we got another new task that should solve this down. For now we can proceed with the current version: https://tokend.atlassian.net/browse/TDV-133